README.md

```diff
@@ -95,7 +95,8 @@ npm install @carbon/charts --save
 | Line        | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark:
 | Curved Line | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark:
 | Pie         | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark:
-| Step        | :white_check_mark: | :white_check_mark: | :white_check_mark: | :hourglass_flowing_sand:
+| Scatter     | :white_check_mark: | :white_check_mark: | :white_check_mark: | :hourglass_flowing_sand:
+| Step        | :white_check_mark: | :hourglass_flowing_sand: | - | -
```

Comment:
Step should be `:white_check_mark:` for all versions now since step charts are just line charts with a different curve in the options https://github.com/carbon-design-system/carbon-charts/blob/1863e140019f598574601e83bd22a937b6fce0ba/packages/react/stories/line-step.stories.js#L52

Reply:
Oups, I think I renamed the wrong line! Thanks for the catch!